src/contrib/persistence/Akka.Persistence.Sql.Common/Settings.cs

```diff
@@ -130,6 +135,7 @@ public SnapshotStoreSettings(Config config)
             SchemaName = config.GetString("schema-name");
             TableName = config.GetString("table-name");
             AutoInitialize = config.GetBoolean("auto-initialize");
+            DefaultSerializer = config.GetString("serializer");
```

Comment:
Even if this setting is not specified by the plugin explicitly, it should still be available. See changes in persistence.conf